server/settings/conversion.py

```diff
@@ -71,10 +71,42 @@ def _convert_validate_version_0_3_3(publish_overrides):
     validate_version["plugin_state_profiles"] = [profile]
 
 
+def _convert_oiio_transcode_0_4_5(publish_overrides):
+    """ExtractOIIOTranscode plugin changed in 0.4.5."""
+    if "ExtractOIIOTranscode" not in publish_overrides:
+        return
+
+    transcode_profiles = publish_overrides["ExtractOIIOTranscode"].get("profiles")
+    if not transcode_profiles:
+        return
+
+    for profile in transcode_profiles:
+        outputs = profile.get("outputs")
+        if outputs is None:
+            return
+
+        for output in outputs :
+            # Already new settings
+            if "display" not in output and "view" not in output:
```

Comment:
I probably made wrong assumption, this was better.
```suggestion
            if "display_view" in output:
```